api/loader/fileloader.go

```diff
@@ -335,6 +318,25 @@ func (fl *fileLoader) Load(path string) ([]byte, error) {
 	return fl.fSys.ReadFile(path)
 }
 
+func loadURL(hc *http.Client, path string) ([]byte, error) {
```

Comment:
The above code in `Load()` was moved here because the linter complained `Load()` had too many nested conditions.